windows/ReactTestApp/MainPage.cpp

```diff
@@ -118,4 +120,52 @@ namespace winrt::ReactTestApp::implementation
         TitleBar().Height(sender.Height());
     }
 
+    void MainPage::SetInitialProperties(
+        std::optional<std::map<std::string, std::any>> const &initialProps)
+    {
+        if (initialProps.has_value()) {
+            ReactRootView().InitialProps([&initialProps](IJSValueWriter const &writer) {
```

Comment:
We should always set `InitialProps` even if it has no value otherwise we won't be able to load the same component with different props since `ReactRootView` won't trigger a reload for the same component name:
```json
{
  "name": "Example",
  "displayName": "Example",
  "components": [
    {
      "appKey": "Example",
      "displayName": "App with no props"
    },
    {
      "appKey": "Example",
      "displayName": "App with some props",
      "initialProperties": {
        "something": "nothing"
      }
    }
  ]
}
```